README.md

```diff
@@ -1,18 +1,25 @@
 # Arcus Templates
 [![Build Status](https://dev.azure.com/codit/Arcus/_apis/build/status/Commit%20builds/CI%20-%20Arcus.Templates?branchName=master)](https://dev.azure.com/codit/Arcus/_build/latest?definitionId=765&branchName=master)
 
-Template repository for new components in Arcus ecosystem.
+.NET Core templates to simplify the new projects and applying best practices.
 
 ![Arcus](https://raw.githubusercontent.com/arcus-azure/arcus/master/media/arcus.png)
 
 # Installation
 
-## Web API
+## Web API Template
+We provide a template to build Web APIs. ([docs](https://templates.arcus-azure.net/features/web-api-template))
 
+First, install the template:
 ```shell
 > dotnet new -i Arcus.Templates.WebApi
 ```
 
+When installed, the template can be created with shortname: arcus-webapi:
+```shell
+> dotnet new arcus-webapi -n Arcus.Demo.WebAPI
```

Comment:
Maybe update the name and remove the output folder in the features `.md` file also so it's in sync.